lightning/src/ln/channel.rs

```diff
@@ -1321,10 +1321,11 @@ impl<Signer: Sign> Channel<Signer> {
 		}
 
 		let total_fee_sat = Channel::<Signer>::commit_tx_fee_sat(feerate_per_kw, included_non_dust_htlcs.len(), self.channel_transaction_parameters.opt_anchors.is_some());
+		let anchors_val = if self.channel_transaction_parameters.opt_anchors.is_some() { ANCHOR_OUTPUT_VALUE_SATOSHI * 2 } else { 0 } as i64;
```

Comment:
one of the anchors might not be there, because one side doesn't have an output and there are no HTLCs?

Reply:
BOLT #3 specifies it this way.